utmap-server/src/app/main/controller/eventController.py

```diff
@@ -25,4 +26,5 @@ def delete(self, _id):
         return deleteOneEvent(_id)
 
     def put(self, _id):
-        pass
\ No newline at end of file
+        data = request.json
+        return updateEvent(_id, data=data)
```

Comment:
I noticed there's a "No newline at end of file" warning around here.  Add an empty line after this one just to be safe.

Reply:
Added